resources/home/dnanexus/utils/io.py

```
@@ -37,16 +46,46 @@ def make_cnv_session(
+    with open(
+        "/home/dnanexus/cnv-template.json", encoding="utf-8", mode="r"
+    ) as fh:
         template = json.load(fh)
 
+    if build == 37:
+        reference_urls = build_37_urls
+    elif build == 38:
+        reference_urls = build_38_urls
+    else:
+        raise RuntimeError(
+            f"Invalid build param given ({build}), must be one of 37 or 38"
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)